src/Meilisearch/TaskEndpoint.cs

```diff
@@ -0,0 +1,107 @@
+namespace Meilisearch
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Net.Http.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// MeiliSearch index to search and manage documents.
+    /// </summary>
+    public class TaskEndpoint
+    {
+        private HttpClient http;
+
+        /// <summary>
+        /// Gets the tasks.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token for this call.</param>
+        /// <returns>Returns a list of the tasks.</returns>
+        public async Task<Result<IEnumerable<TaskInfo>>> GetTasksAsync(CancellationToken cancellationToken = default)
+        {
+            return await this.http.GetFromJsonAsync<Result<IEnumerable<TaskInfo>>>($"/tasks", cancellationToken: cancellationToken)
```

Comment:
I think we do need the interpolation here :)

```suggestion
            return await this.http.GetFromJsonAsync<Result<IEnumerable<TaskInfo>>>("/tasks", cancellationToken: cancellationToken)
```